server/queries/findAdminByCredentials.test.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-undef */
```

Comment:
don't commit empty test files please!

Reply:
Fixed in `jwt-authentication-login-page` branch